### PR TITLE
Basic readme telling how to select symfony version

### PR DESCRIPTION
**Pull request by kejwmen**

### CODE REVIEW

**Comment by dunglas** [comment]
Can you remove it from the generated ZIP archive please? You can do like this: https://github.com/api-platform/api-platform/blob/master/.gitattributes#L21

File: README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+Run `docker-compose up -d`
+
+Choosing symfony version
+------
+
+To choose specific symfony version define `SYMFONY_VERSION` env variable for first run.
+
+Like:
+
+`SYMFONY_VERSION=3.4 docker-compose up -d`
```
Comment (dunglas) [inline README.md]: Can you add a note about the stability flag too?

Comment (kejwmen) [reply README.md]: done. And I noticed we should merge a little fix first (#25)

**Comment by kejwmen** [comment]
#25 was merged, so if it's ok we can merge this one

**Comment by dunglas** [comment]
Thanks @kejwmen! I made some minor modifications.

**Pull request merged**